src/storage/copier/table_copy_utils.cpp

```diff
@@ -370,6 +379,10 @@ bool TableCopyUtils::tryCast(
         int16_t result;
         return StringCastUtils::tryCastToNum(value, length, result);
     }
+    case LogicalTypeID::INT8: {
```

Comment:
Keep this switch. Will be used by union datatype. You don't need to add a test for now